edx_django_utils/monitoring/docs/how_tos/add_code_owner_custom_metric_to_an_ida.rst

```diff
@@ -0,0 +1,46 @@
+Add Code_Owner Custom Metric to an IDA
+======================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+What is the code_owner custom metric?
+-------------------------------------
+
+The code_owner custom metric can be used to create custom dashboards and alerts for monitoring the things that you own. It was originally introduced for the LMS, as is described in this `ADR on monitoring by code owner`_.
+
+You can now easily add this same metric to any IDA so that your dashboards and alerts can work across multiple IDAs at once.
+
+If you want to know about custom metrics in general, see: using_custom_metrics.rst.
+
+.. _ADR on monitoring by code owner: https://github.com/edx/edx-platform/blob/59e0f6efcf2a297806918f8e0020255c1f59ea5f/lms/djangoapps/monitoring/docs/decisions/0001-monitoring-by-code-owner.rst
```

Comment:
Is the use of a commit permalink intentional here? (Less susceptible to bitrot, but might not be the latest version, if there are later clarifications in the ADR.)

Reply:
Unintentional and fixed.